models/interface/account.model.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { Document } from 'mongoose';
+
+export interface IAccount extends Document {

```

Comment:
same here, there should be status field based on some enum

Reply:
added